ooasp/encodings/ooasp_smart_generation.lp

```diff
@@ -50,8 +53,10 @@ assoc_needs_object(ID1, ASSOC, 1..CMIN - N, C2, SIDE, new_object):-
 
 %  EXAMPLE:
 %   We can see the use of this predicate in an example with 9 frames and 1 rack.
-%   Where the need for one more rack since the upperbound of rack is 8 and the number of frames is 9.
-global_ub(C2, 1..((NUM1 + MAX - 1) / MAX)-NUM2, new_object):-
+%   Here, an additional rack is needed since the global upperbound of frames for one rack is 8 but the actual
+%   number of frames is 9.
+global_ub_gap(C2,((NUM1 + MAX - 1) / MAX)-NUM2,new_object):-
```

Comment:
Why were the rules in this file changed again?

Reply:
Because they were in this form in the first PR into main, so they were changed as a part of the conflict resolution, I kept the version from main in this case, as I have assumed this is the newer one, but it would proabably be good if either @susuhahnml or @nrueh could take a look if this is correct.

Reply:
The one with 1... is the correct one. The original version in main had a bug that I fixed with this 1..

Reply:
I reincorporated this in my commit